srcpkgs/devilutionX/template

```diff
@@ -1,16 +1,25 @@
 # Template file for 'devilutionX'
 pkgname=devilutionX
-version=1.2.1
+version=1.4.1
 revision=1
 build_style=cmake
-configure_args="-DVERSION_NUM=$version -DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
-makedepends="SDL2-devel SDL2_ttf-devel SDL2_mixer-devel libsodium-devel"
+configure_args="-DVERSION_NUM=$version -DBINARY_RELEASE=ON -DBUILD_TESTING=OFF -DDISABLE_ZERO_TIER=ON"
+hostmakedepends="git gettext pkg-config smpq"
+makedepends="SDL2-devel SDL2_image-devel fmt-devel libpng-devel libsodium-devel bzip2-devel"
```

Comment:
```suggestion
makedepends="SDL2-devel SDL2_image-devel libpng-devel bzip2-devel
 libsodium-devel fmt-devel zlib-devel asio"
```

Reply:
Some software is downloaded by cmake and statically compiled, there is no need to add it to the template.

Reply:
Bundled source code is used provided the repository does not have them packaged. Your explanation is horribly wrong here.

Reply:
I use the most performative packages and methods, if the author of devilutionX uses X package and indicates it for use, I will use it, even if it is static.